mlflow/R/mlflow/.install-spark.R

```diff
@@ -0,0 +1,9 @@
+parent_dir <- dir("../", full.names = TRUE)
+package <- parent_dir[grepl("mlflow_", parent_dir)]
+install.packages(package)
```

Comment:
Seems this was executed twice.

I prefer the way that put "2.4.5" in a single spark.version file. And then we only change the script to read the version file, instead of split the script into `install-spark
` and another one.  This approach will avoid to change the current package installation order which sometimes matters

Reply:
Makes sense!